proposals/OpaqueReflect.md

```diff
@@ -0,0 +1,132 @@
+# Specification of opaque reflection
+
+## Goal
+
+This PR addresses #1892. The goal is to permit the reflection of functions which contain symbols that are not already in the logic, through the automatic lifting of the symbols which are not in the logic, using measures.
```

Comment:
Another paragraph could be added making explicit why someone would like to have this. It could be a summary of what the referred issue discusses.